src/sqlfluff/dialects/dialect_snowflake.py

```diff
@@ -5736,7 +5736,7 @@ class ExecuteTaskClauseSegment(BaseSegment):
     match_grammar = Sequence(
         "EXECUTE",
         "TASK",
-        Ref("ParameterNameSegment"),
+        Ref("SchemaReferenceSegment"),
```

Comment:
`SchemaReferenceSegment` feels odd here because we're not referring to a schema. Perhaps `ObjectReferenceSegment` might be more appropriate (and I think effectively identical in meaning).

Reply:
Fixed, cheers.